src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -247,7 +248,7 @@ private function createFirewall(ContainerBuilder $container, $id, $firewall, &$a
             $pattern = isset($firewall['pattern']) ? $firewall['pattern'] : null;
             $host = isset($firewall['host']) ? $firewall['host'] : null;
             $methods = isset($firewall['methods']) ? $firewall['methods'] : array();
-            $matcher = $this->createRequestMatcher($container, $pattern, $host, $methods);
+            $matcher = $this->createRequestMatcher($container, $pattern, $host, $firewall['port'] ?? null, $methods);
```

Comment:
this looks for a `port` option on a firewall but it does not exist yet https://github.com/symfony/symfony/blob/e923f801efaf15da94ce5bf037dc036fa46c03d0/src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php#L183

Reply:
If we add the option to firewalls (I think we should but it could be done in another PR, in this case `null` should be passed here) then it should be covered by tests and mentioned in the CHANGELOG as for access_controls

Reply:
So I kept this and I do another P.R. for adding this options to firewall, or I remove all?
Does it finally make sense to put it on for the firewall?

Reply:
the check can't be true, let's remove it (pass `null`) for now and discuss it separately. 
All the existing access_control options are also available at the firewall level, I see no reason for not adding this one

Reply:
Done

Reply:
You should pass null here, the option does not exist.